### ISSUE TITLE
Why is spawn scrap limited to existing scrap?

### ISSUE THREAD

**Issue by KlutzyBubbles** [Toemmsen96/ChatCommands#11]
I am more looking at using this to help with development, and i have been trying to spawn scrap while in company to sell.

The in game dev menu accessed with https://thunderstore.io/c/lethal-company/p/megumin/LethalDevMode/ spawns any item except the scrap information such as value isn't set. I have tried setting the value of the scrap myself but been getting some mixed results.

The chat commands doesn't let me spawn any scrap because im in company.

Basically im trying to ask if you would be able to look into making all scrap spawnable with a scrap value (maybe using the same list the debug menu uses, StartOfRound allItemsList) regardless of what scrap has / can be spawned in any given level.

**Comment by Toemmsen96** [comment]
yes i will

tbh i now looked into the code of LethalDevMode and i dont get what it really does. for that reason i havent yet figured out how that spawns items and stuff.

my guess is it enables a dev mode from the game itself but i will need to find how that works exactly

**Comment by Toemmsen96** [comment]
i am not sure if i can add that, it really seems to be from unity itself

**Comment by Toemmsen96** [comment]
I figured it out and will add the functionality in the next update

**Comment by Toemmsen96** [comment]
This was implemented with Update 2.0.0 in the /spawnitem or /spwitm command. The DebugMenu is now also enabled by default.
